src/components/Editor.js

```diff
@@ -127,9 +128,23 @@ const Editor = React.createClass({
       click: () => this.props.jumpToMappedLocation(sourceLocation)
     };
 
-    showMenu(event, [
+    const menuOptions = [
       jumpLabel
-    ]);
+    ];
+
+    if (this.editor.codeMirror.somethingSelected()) {
+      const watchExpressionLabel = {
```

Comment:
NIT: lets create this label regardless and only do the push if the condition is true